low-level/sensors/impl/generic/raid.py

```diff
@@ -456,6 +456,16 @@ def _send_json_msg(self, alert_type, resource_id, device, drives):
         severity = severity_reader.map_severity(alert_type)
         self._alert_id = self._get_alert_id(epoch_time)
         host_name = socket.getfqdn()
+        if alert_type == self.MISSING:
+            description = "Raid array or drive from raid array is missing."
+        elif alert_type == self.FAULT:
+            description = "Raid array or drive from raid array is faulty."
+        elif alert_type == self.INSERTION:
+            description = "Inserted drive in raid array."
+        elif alert_type == self.FAULT_RESOLVED:
+            description = "Fault for raid drive is resolved"
```

Comment:
use capital RAID eveywhere.

Also here RAID drive is intended?

Reply:
>> use capital RAID eveywhere.
Done.

 fault and fault alert can be for RAID  array level or raid drive level. updated description.